dtfs-central-api/api-tests/v1/utilisation-reports/get-selected-fee-record-details.api-test.ts

```diff
@@ -13,25 +14,54 @@ import { testApi } from '../../test-api';
 import { SqlDbHelper } from '../../sql-db-helper';
 import { wipe } from '../../wipeDB';
 import { mongoDbClient } from '../../../src/drivers/db-client';
-import { aBank } from '../../../test-helpers/test-data/bank';
-import { aReportPeriod } from '../../../test-helpers/test-data/report-period';
+import { aBank, aReportPeriod } from '../../../test-helpers/test-data';
 
 const getUrl = (reportId: number | string) => `/v1/utilisation-reports/${reportId}/selected-fee-records-details`;
 
 interface CustomResponse extends Response {
   body: SelectedFeeRecordDetails;
 }
 
+console.error = jest.fn();
+
 describe('GET /v1/utilisation-reports/:id/selected-fee-records-details', () => {
   const bankId = '123';
   const bank: Bank = { ...aBank(), id: bankId, name: 'Test bank' };
 
   const reportId = 1;
+
+  const facilityUtilisationData = FacilityUtilisationDataEntityMockBuilder.forId('000123').build();
+
   const reportPeriod = aReportPeriod();
+  const utilisationReport = UtilisationReportEntityMockBuilder.forStatus('RECONCILIATION_IN_PROGRESS')
```

Comment:
I think things have gotten slightly messed up merging or rebasing here as I think the set up for these was moved into the tests

Reply:
Hm very possibly. I'll try to set it back to what is currently on main